src/Tizen.NUI.Components/Controls/DatePicker.cs

```diff
@@ -0,0 +1,323 @@
+        }
+
+        /// <summary>
+        /// An event emitted when DatePicker value changed, user can subscribe or unsubscribe to this event handler.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<DateChangedEventArgs> DateChanged;
+        
+        /// <summary>
+        /// The month value of DatePicker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int Month
+        {
+            get
+            {
+                return month;
+            }
+            set
+            {
+                if (value < 1 || value > 12 || value == month) return;
+
+                month = value;
+                monthPicker.CurrentValue = month;
+            }
+        }
+
+        /// <summary>
+        /// The day value of DatePicker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int Day
+        {
+            get
+            {
+                return day;
+            }
+            set
+            {
+                if (value < 1 || value > 31 || value == day) return;
+
+                day = value;
+                dayPicker.CurrentValue = day;
+            }
+        }
+
+        /// <summary>
+        /// The year value of DatePicker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int Year
+        {
+            get
+            {
+                return year;
+            }
+            set
+            {
+                if (value < 1970 || value > 2038 || value == year) return;
+
+                year = value;
+                yearPicker.CurrentValue = year;
+            }
+        }
+
+                /// <summary>
+        /// Initialize TimePicker object.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override void OnInitialize()
+        {
+            base.OnInitialize();
+
+            dayPicker = new Picker()
+            {
+                MinValue = 1,
+                MaxValue = 31,
+            };
+            dayPicker.ValueChanged += OnDayValueChanged;
+
+            monthPicker = new Picker()
+            {
+                MinValue = 1,
+                MaxValue = 12,
+            };
+            monthPicker.ValueChanged += OnMonthValueChanged;
+
+            yearPicker = new Picker()
+            {
+                MinValue = 1970,
+                MaxValue = 2038,
+            };
+            yearPicker.ValueChanged += OnYearValueChanged;
+
+            //FIXME: Changes it to get current system date.
+            day = 1;
+            month = 1;
+            year = 1970;
+        }
+    
+        private void Initialize()
+        {
+            HeightSpecification = LayoutParamPolicies.MatchParent;
+
+            Layout = new LinearLayout() { 
+                LinearOrientation = LinearLayout.Orientation.Horizontal,
+                CellPadding = new Size(50, 339),
```

Comment:
Could you add ApplyStyle(DatePickerStyle style) and apply DefaultThemeCommon.cs's CellPadding in ApplyStyle?

Reply:
done